HW_3/TASK20.py

```diff
@@ -0,0 +1,42 @@
+# Напишите программу, которая вычисляет стоимость введенного пользователем слова.
+# Будем считать, что на вход подается только одно слово,
+# которое содержит либо только английские, либо только русские буквы.
+# В случае с английским алфавитом очки распределяются так:
+# A, E, I, O, U, L, N, S, T, R – 1 очко;
+# D, G – 2 очка;
+# B, C, M, P – 3 очка;
+# F, H, V, W, Y – 4 очка;
+# K – 5 очков;
+# J, X – 8 очков;
+# Q, Z – 10 очков.
+# А русские буквы оцениваются так:
+# А, В, Е, И, Н, О, Р, С, Т – 1 очко;
+# Д, К, Л, М, П, У – 2 очка;
+# Б, Г, Ё, Ь, Я – 3 очка;
+# Й, Ы – 4 очка;
+# Ж, З, Х, Ц, Ч – 5 очков;
+# Ш, Э, Ю – 8 очков;
+# Ф, Щ, Ъ – 10 очков.
+
+word = input("Введите слово: ")
+summa = 0
+if word.isalpha():
+    for letter in word:
+        match letter.upper():
+            case "A" | "E" | "I" | "O" | "U" | "L" | "N" | "S" | "T" | "R": summa += 1
+            case "А" | "В" | "Е" | "И" | "Н" | "О" | "Р" | "С" | "Т":       summa += 1
+            case "D" | "G":                                                 summa += 2
+            case "Д" | "К" | "Л" | "М" | "П" | "У":                         summa += 2
+            case "B" | "C" | "M" | "P":                                     summa += 3
+            case "Б" | "Г" | "Ё" | "Ь" | "Я":                               summa += 3
+            case "F" | "H" | "V" | "W" | "Y":                               summa += 4
+            case "Й" | "Ы":                                                 summa += 4
+            case "K":                                                       summa += 5
+            case "Ж" | "З" | "Х" | "Ц" | "Ч":                               summa += 5
+            case "J" | "X":                                                 summa += 8
+            case "Ш" | "Э" | "Ю":                                           summa += 8
+            case "Q" | "Z":                                                 summa += 10
+            case "Ф" | "Щ" | "Ъ":                                           summa += 10
```

Comment:
Такое расположение кода хоть и допускается, но не рекомендуется. 
```suggestion
            case "A" | "E" | "I" | "O" | "U" | "L" | "N" | "S" | "T" | "R":
                summa += 1
            case "А" | "В" | "Е" | "И" | "Н" | "О" | "Р" | "С" | "Т":
                summa += 1
            case "D" | "G":
                summa += 2
            case "Д" | "К" | "Л" | "М" | "П" | "У":
                summa += 2
            case "B" | "C" | "M" | "P":
                summa += 3
            case "Б" | "Г" | "Ё" | "Ь" | "Я":
                summa += 3
            case "F" | "H" | "V" | "W" | "Y":
                summa += 4
            case "Й" | "Ы":
                summa += 4
            case "K":
                summa += 5
            case "Ж" | "З" | "Х" | "Ц" | "Ч":
                summa += 5
            case "J" | "X":
                summa += 8
            case "Ш" | "Э" | "Ю":
                summa += 8
            case "Q" | "Z":
                summa += 10
            case "Ф" | "Щ" | "Ъ":
                summa += 10
```

Reply:
Как то на семинаре я говорил, почему не стоит использовать битовые операторы.

Reply:
Сергей, подумайте, как решить при помощи словарей, уверяю вас, решение будет намного проще.